static/app/views/dashboardsV2/widgetBuilder/buildSteps/filterResultsStep/issuesSearchBar.tsx

```diff
@@ -57,7 +57,9 @@ function IssuesSearchBar({
   );
 }
 
-export default withIssueTags(IssuesSearchBar);
+const IssuesSearchBar = withIssueTags(IssuesSearchBarContainer);
+
+export {IssuesSearchBar};
```

Comment:
Thank you. I wasn't quite sure how to do the named export with the HoC so this is good. I realized the tests are failing elsewhere because this `IssuesSearchBar` is shared with the modal, we need to update the import in `issueWidgetQueriesForm.tsx`. I'll push a commit up

Reply:
I think you can also do `export const IssuesSearchBar = withIssueTags(IssuesSearchBarContainer);` 😄